packages/components/src/popover/index.js

```diff
@@ -199,6 +200,22 @@ function setClass( element, name, toggle ) {
 	}
 }
 
+function useFocusOutsideWithRef( ref, onFocusOutside ) {
```

Comment:
To be honest,  I feel useFocusOutside should use Dom events by default, that said, when we tried that recently, we had an issue with "nested popovers" and "nested modals" and we didn't really found a way to solve it without DOM events.

Reply:
I believe the change in this PR is probably going to break some of these "nested popovers"  behaviors.

Reply:
@youknowriad Do you know which part of the code is responsible for focusing on the toolbar when pressing shift + tab? I'm curious why the toolbar behavior isn't affected in that case, only when using alt + F10.

Reply:
Not on top of my head, maybe @ellatrix knows

Reply:
That should be writing flow => focus capture elements. Focus is transferred to the focus capture element and then the browser focusses the next element in the DOM.